src/context/WalletProvider/KeepKey/components/Connect.tsx

```diff
@@ -65,6 +67,7 @@ export const KeepKeyConnect = () => {
           return wallet
         } catch (e) {
           const secondAdapter = await getAdapter(KeyManager.KeepKey, 1)
+          // @ts-ignore TODO(gomes): FIXME, most likely borked because of WebUSBKeepKeyAdapter
```

Comment:
WebUSBKeepKeyAdapter types are too borked, fiddled for two hours in hdwallet with no success here, leaving this as-is for now - there is no effective runtime change here

Reply:
Keeping it as-is for now, the types refactor required to make this work would be a lot more complex, rage quitted after ~2 hours